src/xdocs/beginning_development.xml

```diff
@@ -49,6 +49,12 @@
         <a href="idea.html">IntelliJ IDEA IDE</a><br/>
       </p>
 
+      <p>
+        Video walk through of these instructions is available in this
```

Comment:
`instructions is available` => `instructions are available`

Reply:
`in this` => `in {your name here}`

Replace {your name here} with the name you go by on your Youtube channel.

Reply:
Done